dom.bs

```diff
@@ -1186,33 +1187,70 @@ for discussion).
 
  <li><p>Initialize <var>event</var>'s {{Event/currentTarget}} attribute to <var>object</var>.
 
+ <li><p><a>Find</a> listener type matches for <var>object</var>.
+
+  <li>
+   <p>If <var>event</var>'s <a>found listener flag</a> is unset, run
+   these substeps:
+
+   <ol>
+    <li><p>Set <var>original event type</var> to <var>event</var>'s {{Event/type}} attribute value.
+
+    <li>
+     <p>If <var>event</var>'s {{Event/type}} attribute value is an <a>ASCII case-insensitive</a>
+     match for any of the strings in the first column in the following table, set <var>event</var>'s
+     {{Event/type}} attribute value to the string in the second column on the same row as the
+     matching string, otherwise, terminate these substeps.
+
+     <table>
+      <thead>
+       <tr><th>Event Type<th>Legacy Event Type
+      <tbody>
+       <tr><td>"<code>animationend</code>"<td>"<code>webkitAnimationEnd</code>"
+       <tr><td>"<code>animationiteration</code>"<td>"<code>webkitAnimationIteration</code>"
+       <tr><td>"<code>animationstart</code>"<td>"<code>webkitAnimationStart</code>"
+       <tr><td>"<code>transitionend</code>"<td>"<code>webkitTransitionEnd</code>"
+     </table>
+
+     <li><p><a>Find</a> listener type matches for <var>object</var>.
+
+     <li><p>Set <var>event</var>'s {{Event/type}} attribute value to <var>original event type</var>.
+   </ol>
+
+ <li><p>Unset <var>event</var>'s <a>found listener flag</a> if set.
+</ol>
+
+<p>To <dfn export for="event listener" id=concept-event-listener-find>find</dfn> listener type
```

Comment:
Not really married to this "find listener type matches for object" name, suggested changes welcome.